scrapers/CorbinFisher.yml

```diff
@@ -0,0 +1,28 @@
+name: corbinfisher
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - corbinfisher.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1
+      Date:
+        selector: //div[@class="added"]
+        postProcess:
+          - replace:
+              - regex: (^\S+\s+)(.+\d{4})(.+)
+                with: $2
+          - parseDate: January 2, 2006
+      Performers:
+        Name: //div[@class="modelFeaturing"]/a | //div[@class="modelFeaturing"]//a
+      Details:
+        selector: //div[@class="description"]/p
```

Comment:
you forgot the concat for cases like `https://www.corbinfisher.com/tour/trailers/acm-william-fills-up-quinn.html`
```yaml
        selector: //div[@class="description"]/p
        concat: "\n\n"
```

Reply:
Yeah I dunno why I had that union there as its not needed.
All fixed now. :-)